docs/actions/prepare_shop.md

```diff
@@ -1,36 +1,11 @@
 # prepare_shop
-This action checks out the SDK and the CE shop. If this is a pull request, it is merged into the base ref.
 
-After this, "make setup" and "make addbasicservices" is run. If there are additional modules specified in
-the "add_services" input, they are loaded as well. The actioon also takes care of changing the ports for
-apache, if the "nginx-rp" service is installed.
+This action checks out the SDK. It then creates a composer.json based on a template
+or contained in a given repository. In a pull request event, the base ref is checked out and the given PR is merged into it.
 
-Afterwards, the variables PHP_VERSION and MYSQL_VERSION are set in the ".env" file and the inputs "custom_ini_xdebug" and "custom_ini_error_reporting" are used to update containers/php/custom.ini and we substitute /var/www/ by /var/www/source/ in "containers/httpd/project.conf".
+The action uses the Makefile fo the SDK to start the containers and composer to install the packages.
```

Comment:
Small typo:
`The action uses the Makefile for the SDK to start the containers and composer to install the packages.`